queries/projects/lup-projects.sql

```diff
@@ -25,7 +31,11 @@ WITH lups_project_assignments AS (
   INNER JOIN -- inner because not all projects should be visible to users based on visibility field in "where" clause
     dcp_project AS p ON lup.dcp_project = p.dcp_projectid
   INNER JOIN -- inner because we only want certain milestones with the status included in the "where" clause
-    dcp_projectmilestone AS mm ON lup.dcp_project = mm.dcp_project
+    (SELECT * FROM dcp_projectmilestone WHERE statuscode <> 'Overridden') AS mm ON lup.dcp_project = mm.dcp_project
+  LEFT JOIN
+    dcp_communityboarddisposition AS disp
+    ON disp.dcp_project = lup.dcp_project
+    AND disp.dcp_recommendationsubmittedby = '${id:value}' -- plugs in contactid
```

Comment:
What do lines 35-38 do? Just asking to learn/understand

Reply:
I had to join the `dcp_communityboarddisposition` table onto the LUP project assignment results in order to find out if the LUP had submitted their recommendation yet (see lines 8-10). In order to join the dispositions onto the LUP project assignment, I needed to join using the project id and contact id. Now that I'm looking at this again though, I think I may also need to include `dcp_lupteammemberrole` and the corresponding "representing" field in the disposition record to the join criteria, so we don't accidentally think a BB review is done when the BP has only submitted the BP review not BB yet.

Reply:
@godfreyyeung updated now FYI